inlong-dashboard/src/ui/pages/ClusterTags/ClusterBindModal.tsx

```diff
@@ -79,7 +80,9 @@ const Comp: React.FC<Props> = ({ clusterTag, ...modalProps }) => {
               formatResult: result =>
                 result?.list?.map(item => ({
                   ...item,
-                  label: `${item.name} (${item.type})`,
+                  label: `${item.displayName} (${
```

Comment:
Can you make a compatibility here: when `DisplayName` does not exist, display` name`. Avoid the `null` in the screenshot.